cdbot/cogs/cyber.py

```diff
@@ -346,6 +346,23 @@ def __init__(self, bot: Bot):
 
         await self.countdown('5th March 2019', 'CyberStart Essentials', ctx)
 
+    @command()
+    async def hundered(self, ctx: Context):
+        """
+        Gets the number of 100% and true 100% users
+        """
+
+        game = true = 0
+        for member in ctx.guild.members:
+            for role in member.role:
+                if role.name == '100 Percent':
+                    game += 1
+                elif role.name == 'True 100%':
```

Comment:
By using `elif` instead of a separate `if` statement, you count members with True 100% separately. More importantly, every member with the True 100% role will also have the 100% role, meaning the output for True 100% will always be 0.

Reply:
That's actually not true because it iterates over all their roles. The _is_ a problem though in that snippet that my shourly disapear form a push.

Reply:
Nevermind.